packages/components/forms/stories/Formik.stories.tsx

```diff
@@ -0,0 +1,337 @@
+    >
+      {({ handleSubmit }) => (
+        <Form onSubmit={handleSubmit}>
+          <Field name="cities">
+            {({ field }) => (
+              <FormControl>
+                <FormControl.Label>City</FormControl.Label>
+
+                <Select {...field}>
+                  <Select.Option value="Cape Town">Cape Town</Select.Option>
+                  <Select.Option value="Mumbai">Mumbai</Select.Option>
+                  <Select.Option value="Rio de Janeiro">
+                    Rio de Janeiro
+                  </Select.Option>
+                </Select>
+              </FormControl>
+            )}
+          </Field>
+
+          <Button variant="primary" type="submit">
+            Submit
+          </Button>
+        </Form>
+      )}
+    </Formik>
+  );
+};
+
+export const WithRadioGroup = () => {
+  const onSubmit = (data) => action('submitted data')(data);
+
+  return (
+    <Formik
+      initialValues={{
+        fruit: 'apples',
+      }}
+      onSubmit={onSubmit}
+    >
+      {({ handleSubmit }) => (
+        <Form onSubmit={handleSubmit}>
+          <Field name="fruit">
+            {({ field }) => (
+              <FormControl>
+                <FormControl.Label>Fruits</FormControl.Label>
+
+                <RadioGroup {...field}>
+                  <Radio id="apples" value="apples">
+                    Apples
+                  </Radio>
+                  <Radio id="pears" value="pears">
+                    Pears
+                  </Radio>
+                  <Radio id="peaches" value="peaches">
+                    Peaches
+                  </Radio>
+                </RadioGroup>
+              </FormControl>
+            )}
+          </Field>
+
+          <Button variant="primary" type="submit">
+            Submit
+          </Button>
+        </Form>
+      )}
+    </Formik>
+  );
+};
+
+export const WithCheckboxGroup = () => {
+  const onSubmit = (data) => action('submitted data')(data);
+
+  return (
+    <Formik
+      initialValues={{
+        fruits: ['apples'],
+      }}
+      onSubmit={onSubmit}
+    >
+      {({ handleSubmit }) => (
+        <Form onSubmit={handleSubmit}>
+          <Field name="fruits">
+            {({ field }) => (
+              <FormControl>
+                <FormControl.Label>Fruits</FormControl.Label>
+
+                <CheckboxGroup {...field}>
+                  <Checkbox id="apples" value="apples">
+                    Apples
+                  </Checkbox>
+                  <Checkbox id="pears" value="pears">
+                    Pears
+                  </Checkbox>
+                  <Checkbox id="peaches" value="peaches">
+                    Peaches
+                  </Checkbox>
+                </CheckboxGroup>
+              </FormControl>
+            )}
+          </Field>
+
+          <Button variant="primary" type="submit">
+            Submit
+          </Button>
+        </Form>
+      )}
+    </Formik>
+  );
+};
+
+export const WithUseFormik = () => {
```

Comment:
I added this example so we can show that it also works with this approach, but they recommend using the `<Formik>` and `<Field>` components